sources/tbb/machine/linux_ia64.h

```diff
@@ -21,7 +21,9 @@
 #define __TBB_machine_linux_ia64_H
 
 #include <stdint.h>
-#include <ia64intrin.h>
+#if (CC_PLATFORM != CC_PLATFORM_NX)
+    #include <ia64intrin.h>
+#endif

```

Comment:
I don't think we should modify tbb source code, just not use tbb on nx.

Reply:
yes i will change it
I used it when there was no nvn using vukan before. Now things have changed.